backend/sites/src/app/entities/profileSubmissions.entity.ts

```diff
@@ -32,15 +32,15 @@ export class ProfileSubmissions {
   @Column('bigint', { primary: true, name: 'sprof_site_id' })
   sprofSiteId: string;
 
-  @ManyToOne(
-    () => SiteProfiles,
-    (siteProfiles) => siteProfiles.profileSubmissions,
-  )
-  @JoinColumn([
-    { name: 'sprof_site_id', referencedColumnName: 'siteId' },
-    { name: 'sprof_date_completed', referencedColumnName: 'dateCompleted' },
-  ])
-  siteProfiles: SiteProfiles;
+  // @ManyToOne(
+  //   () => SiteProfiles,
+  //   (siteProfiles) => siteProfiles.profileSubmissions,
```

Comment:
Remove the commented code

Reply:
need this for future in case if there will be conflict while data sync